docs/user/data_management.rst

```diff
@@ -7,27 +7,38 @@ distribution and migrate data between tables or environments.
 
 .. note::
 
-    Currently this section applies only to the Accumulo Data Store.
+    Currently this chapter applies only to the Accumulo Data Store.
+
+.. _index_structure:
 
 Index Structure
 ---------------
 
-By default, GeoMesa creates three indices:
-
-- Z2 - the Z2 index uses a z-order curve to index latitude and longitude. This is the primary index used
-  to answer queries with a spatial component but no temporal component.
-- Z3 - the Z3 index uses a z-order curve to index latitude, longitude and time. This is the primary index
-  used to answer queries with both a spatial and temporal component.
-- Record - the record index stores features by feature ID. It is used for any query by ID. Additionally,
+By default, GeoMesa creates a number of indices:
+
+- **Z2** - the Z2 index uses a Z-order curve to index latitude and longitude. This is the primary index used
```

Comment:
could we reword these a bit? i find the double 'with' slightly confusing. maybe something like:
the Z2 index uses a two-dimensional Z-order curve to index latitude and longitude for point data. This index will only be created if your schema has the geometry type `Point`. It can be used to efficiently answer queries with a spatial component.
etc.